tls/s2n_config.c

```diff
@@ -447,7 +507,8 @@ int s2n_config_add_cert_chain_and_key_with_status(struct s2n_config *config, con
 
 int s2n_config_add_cert_chain_and_key(struct s2n_config *config, const char *cert_chain_pem, const char *private_key_pem)
 {
-    GUARD(s2n_config_add_cert_chain_and_key_with_status(config, cert_chain_pem, private_key_pem, NULL, 0));
+    GUARD(s2n_config_add_cert_chain_and_key_with_extensions(config, cert_chain_pem, private_key_pem,
```

Comment:
Autoformat? Looks like this line was split at 100 chars, we usually aim for 120.